modules/wowchemy/layouts/partials/blocks/people.html

```diff
@@ -0,0 +1,66 @@
+{{/* Wowchemy Blocks: People */}}
+{{/* Documentation: https://wowchemy.com/blocks/ */}}
+{{/* License: https://github.com/wowchemy/wowchemy-hugo-themes/blob/main/LICENSE.md */}}
+
+{{/* Initialise */}}
+{{ $page := .wcPage }}
+{{ $block := .wcBlock }}
+{{ $show_social := $block.design.show_social | default false }}
+{{ $show_interests := $block.design.show_interests | default true }}
+{{ $show_organizations := $block.design.show_organizations | default false }}
+{{ $show_role := $block.design.show_role | default true }}
+
+<div class="row justify-content-center people-widget">
+  {{ with $block.content.title }}
+  <div class="col-md-12 section-heading">
+    <h1>{{ . | markdownify | emojify }}</h1>
+    {{ if $block.content.subtitle }}<p>{{ $block.content.subtitle | markdownify | emojify }}</p>{{ end }}
```

Comment:
My bad, I just copied the old v1 widget without noticing these changes.
In the v1 version, the `people` widget did not inherit the heading from the common code in `parse_block_v1`:

https://github.com/wowchemy/wowchemy-hugo-themes/blob/b9752d697a7bb94de3d085db3988606e138f6657/modules/wowchemy/layouts/partials/functions/parse_block_v1.html#L111

I think this is due to the fact that this widget was not intended to be use with a 2 columns layout. I tried to modify the code, and the result for the 2 columns layout is the following:

![people-2-columns](https://user-images.githubusercontent.com/43784886/217489106-b841e55f-2210-4796-9db0-107f15d5e845.png)

which I personally do not really like due to the huge blank space on the left.

I can easily achieve both solutions (either the common block heading with the possibility of 2 columns layout or the specific one inside `people.html`, with only the possibility of 1 column): tell me what you think would be the best one.

Reply:
OK, let's just update the HTML and styles for the single column design in the PR so that they are consistent with the one column wrapper in that function, as per the first comment above.